.vscode/launch.json

```diff
@@ -5,30 +5,14 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "integration-tests - core/availability-check/opportunity-full",
+            "name": "integration-tests - core/common-error-conditions/incomplete-customer-details",
```

Comment:
Constantly editing this configuration is not ideal. Ideally, we'd have a "Integration Test this file", which is possible with `${file}` (https://github.com/microsoft/vscode-recipes/tree/master/debugging-mocha-tests#configure-launchjson-file-for-your-test-framework) but that wouldn't work here as it'd give the full path including `packages/openactive-integration-tests`.

Not very important though

Reply:
I think Jest can handle the full path?